extensions/prototype-content-with-issues-table/get_usage.R

```diff
@@ -16,13 +16,9 @@ usage_dtype <- tibble::tibble(
 
 # A rough implementation of how a new firehose usage function would work in
 # `connectapi`.
-get_usage_firehose <- function(client, from = NULL, to = NULL) {
+get_usage_firehose <- function(client) {
```

Comment:
`from` and `to` parameters removed in response to:

> [Here, but also elsewhere in these functions: you've got from/to, but are you using those anywhere here?](https://github.com/posit-dev/connect-extensions/pull/33#discussion_r1989924592)

Reply:
Thanks!